ui/src/components/yaml/styles.module.less

```diff
@@ -324,17 +324,14 @@
       }
     }
 
```

Comment:
The `width` property remains unchanged at `400px`. Ensure this value is still appropriate for the component's layout.